NiL.JS/ExportTable.cs

```diff
@@ -23,12 +23,52 @@ public sealed class ExportTable : IEnumerable<KeyValuePair<string, JSValue>>
 
                 return result;
             }
-            internal set
+             set
             {
                 _items[key] = value;
             }
         }
+        /// <summary>
+        /// Add object constructor to export 
+        /// </summary>
+        /// <remarks>We dont recommend to use, because api can change and its can be broken in future. Use indexers</remarks>
+        /// <param name="type">Class</param>
+        /// <param name="name"></param>
+        public void DefineConstructor(Type type, string name)
+        {
+            var ctor = GlobalContext.CurrentGlobalContext.GetConstructor(type);
```

Comment:
GlobalContext.Current* is not safe. Usage of it can bring to concurrency errors. There are two way to improve this code:
1. Add `Context` to constructor of `ExportTable`, pass the `Context` of the `Module` and use it here
2. Add `Context` to parameters of this method. In this case it might be just an extension method

Reply:
> GlobalContext.Current* is not safe. Usage of it can bring to concurrency errors. There are two way to improve this code:
> 
>     1. Add `Context` to constructor of `ExportTable`, pass the `Context` of the `Module` and use it here
> 
>     2. Add `Context` to parameters of this method. In this case it might be just an extension method

I do first